tools/importers/helpers/fileReader.ts

```diff
@@ -0,0 +1,59 @@
+const modifiers = require('./modifiers');
+
+const recursiveTokenReader = (obj, keyResult: string, result: Array<any>, importerVariables) => {
```

Comment:
do we need to pass `result` as a parameter? Can't we merge results at the end of this function?

Reply:
Fixed, parameter removed.